packages/plugins/mocha-tests/src/lib/index.js

```diff
@@ -61,9 +62,27 @@ class MochaTestRunner {
           },
           (contracts, next) => {
             for(const contract of contracts) {
-              const instance = compiledContracts[contract.className];
-              const contractObj = new web3.eth.Contract(instance.abiDefinition, contract.deployedAddress);
-              Object.setPrototypeOf(compiledContracts[contract.className], contractObj);
+              const instance = EmbarkJS.Blockchain.Contract(contract);
```

Comment:
Does EmbarkJS' contract still use the blockchainConnector? If so, this will probably fail. If not, disregard this comment

Reply:
I tested this with my tiny demo app and it worked.